Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -1050,11 +1058,14 @@ public void ProcessSynchronousEvents(bool forceProcess = false)
                     }
                 }
 
-                //Sample the portfolio value over time for chart.
-                SampleEquity(time, Math.Round(_algorithm.Portfolio.TotalPortfolioValue, 4));
+                if (canSampleEquity)
```

Comment:
The aim was to not update real time price; not stop sampling. Stopping sampling will stop chart movement as well and make the chart not reflect market closed periods (pros and cons of that; so for now lets keep current behavior of showing market closed periods).